pkg/hubble/dropeventemitter/dropeventemitter.go

```diff
@@ -19,15 +19,17 @@ import (
 	slimv1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
 	metaslimv1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	slimscheme "github.com/cilium/cilium/pkg/k8s/slim/k8s/client/clientset/versioned/scheme"
+	"github.com/cilium/cilium/pkg/k8s/watchers"
 	"github.com/cilium/cilium/pkg/time"
 )
 
 type DropEventEmitter struct {
-	reasons  []string
-	recorder record.EventRecorder
+	reasons    []string
+	recorder   record.EventRecorder
+	K8sWatcher watchers.CacheAccessK8SWatcher
```

Comment:
Why does this need to be public?

Reply:
Good catch. It doesn't need to be. Converting it to private.